packages/core/src/Site/index.js

```diff
@@ -712,6 +712,7 @@ class Site {
     return new Promise((resolve, reject) => {
```

Comment:
are the changes here still needed? 😮

Reply:
Sorry, these are from a debug session, I forgot to remove them. 😓

Reply:
no worries 😁